src/gui/GuiRunner.hh

```diff
@@ -43,9 +43,7 @@ class IGNITION_GAZEBO_GUI_VISIBLE GuiRunner : public QObject
 
   /// \brief Constructor
   /// \param[in] _worldName World name.
-  /// \todo Move to src/gui on v6.
-  public: explicit IGN_DEPRECATED(5.0) GuiRunner(
-      const std::string &_worldName);
+  public: explicit GuiRunner(const std::string &_worldName);
```

Comment:
moving this header file has broken inn-launch:

* https://github.com/ignitionrobotics/ign-launch/blob/main/plugins/gazebo_gui/GazeboGui.cc#L23

Reply:
https://github.com/ignitionrobotics/ign-launch/issues/118

Reply:
hummmmm weird https://github.com/ignitionrobotics/ign-launch/pull/119